packages/opds-web-client/src/components/__tests__/Breadcrumbs-test.tsx

```diff
@@ -30,12 +30,29 @@ describe("Breadcrumbs", () => {
 
     let list = wrapper.find("ol");
     let links = wrapper.find(CatalogLink);
+    let currentLink = wrapper.find("ol li").last();
     expect(list.hasClass("breadcrumb")).to.equal(true);
-    expect(links.length).to.equal(2);
+    // The current page is no longer a link unless `currentLink` is true:
+    expect(links.length).to.equal(1);
     expect(links.at(0).props().children).to.contain("2nd title");
     expect(links.at(0).props().collectionUrl).to.equal("2nd url");
     // last link is wrapped in <strong>
-    expect((links.at(1).props().children as any).props.children).to.contain("last title");
+    expect(currentLink.text()).to.equal("last title");
+  });
+
+  it("shows links with bootstrap classes", () => {
```

Comment:
I would rename these two tests to reflect that they're testing with and without the current link now.